tools/ci_build/github/azure-pipelines/orttraining-1p-linux-gpu-ci-pipeline.yml

```diff
@@ -0,0 +1,69 @@
+parameters:
+  - name: build_flag
+    displayName: Build Flag
+    type: string
+    default: --enable_onnx_tests
+  - name: image_tag
+    displayName: Image Tag
+    type: string
+    default: latest
+
+stages:
+- stage: Python_Packaging
+
+  variables:
+    docker_image_prefix: onnxruntime-training
+    linux_gpu_dockerfile: ../../../../../dockerfiles/Dockerfile.training
```

Comment:
$(Build.SourcesDirectory)/dockerfiles/Dockerfile.training

Reply:
Changed

Reply:
linux_gpu_dockerfile is relative to the source directory inside docker container